src/app/lib/utilities/getRemoteConfig/index.test.js

```diff
@@ -0,0 +1,104 @@
+import fetchMock from 'fetch-mock';
+import { CONFIG_REQUEST_RECEIVED, CONFIG_FETCH_ERROR } from '#lib/logger.const';
+
+const mockUrl =
+  'https://mock-config-endpoint?application=simorgh&service=mundo&__amp_source_origin=http://localhost';
+const mockResponse = { testToggle: { enabled: true } };
+fetchMock.mock(mockUrl, mockResponse);
+
+describe('getRemoteConfig', () => {
+  beforeEach(async () => {
+    jest.resetModules();
+    await import('#testHelpers/loggerMock');
+    process.env.SIMORGH_CONFIG_URL = 'https://mock-config-endpoint';
+  });
+
+  it('should return null if enableFetchingToggles is not enabled', async () => {
+    jest.mock('#lib/config/toggles', () => ({
+      local: {
+        enableFetchingToggles: { enabled: false },
+      },
+    }));
+
+    // Dynamic import is used in these tests so the toggles file values can be changed
+    const getRemoteConfig = await import('.');
+    const remoteToggles = await getRemoteConfig.default('mundo');
+
+    expect(remoteToggles).toBe(null);
```

Comment:
Do we need to explicit with value checking? Just in case we decide to change this to be `undefined` for example, could we use `toBeFalsy()` so we don't have to go back into these tests to change them?

Reply:
These tests have changed so this so this check no longer happens. 👍